build/post-integration-test-docker.sh

```diff
@@ -21,9 +32,9 @@ rm -rf ${it_results} 2>/dev/null
 mkdir -p ${it_results}/server-logs
 mkdir -p ${it_results}/fhir-server-test
 
-containerId=$(docker ps -a | grep fhir | cut -d ' ' -f 1)
-if [[ -z "${containerId}" ]]; then
-    echo "Warning: Could not find fhir container!!!"
+containerId=$(docker ps -a | grep ${1}[-_]fhir-server | cut -d ' ' -f 1)
```

Comment:
some common functions would be good for this kind of thing (I'm assuming this isn't the only place we need something like this.

Reply:
agree

Reply:
I added it under common with a filename of `gather-pretest-logs.sh`.  I also renamed the existing `gather-logs.sh` to `gather-posttest-logs.sh` to avoid confusion